### ISSUE TITLE
angular.min.js and angular-animate.min.js are not included

### ISSUE THREAD

**Issue by Danieler** [nikhilmodak/gulp-ngdocs#100]
the angular.min.js and angular-animate.min.js are not included in the generated folder
      var gulpDocs = require('gulp-ngdocs'),
            gulp = require('gulp');

           gulp.task('ngdocs', [], function () {

    return gulp.src('app/**/*.js')
        .pipe(gulpDocs.process())
        .pipe(gulp.dest('./docs'));
     });

**Comment by mohammadwali** [comment]
Any update on this ? :(

**Comment by jongunter** [comment]
This suddenly started happening to me today. I'm trying to figure out what change in my code triggered it.

**Comment by racon-am** [comment]
Any solution for this problem?

**Comment by huerlisi** [comment]
Could you test with the newest release, pls? I think this could be related to npm 3. Support for that was added in c96abaf927 and a new release has been made a couple days ago.

**Comment by andyperlitch** [comment]
Yes, I cut a new release which contains quite a few merged pull requests. @Danieler: for now, list the dep version as v0.2.13 in your package.json and keep using that version while the latest stabilizes. I'll do what I can on this project but I no longer use it in my projects and am busier than i like to admit to myself.
